app_connections/controllers/comments.controller.js

```diff
@@ -23,3 +28,14 @@ exports.postComment = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+  const comment_id = req.params.comment_id;
+  deleteCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
You can use res.sendStaus(204) if you only want to send the status without any content